tasks/build/package_scripts/pre_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if command -v systemctl >/dev/null && systemctl is-active kibana.service >/dev/null; then
+    systemctl --no-reload stop kibana.service
+fi
+if [ -x /etc/init.d/kibana ]; then
```

Comment:
I believe we addressed systemctl in another comment.  

Checking for init.d/kibana is useful because all of those commands end up executing from that file, if it doesn't exist we want to avoid errors.